src/app/client/src/app/modules/workspace/components/data-driven/data-driven.component.ts

```diff
@@ -316,12 +317,13 @@ export class DataDrivenComponent extends WorkSpace implements OnInit, OnDestroy,
   /**
     * Issue #SB-1448,  If previous url is not from create page, redirect current page to 'workspace/content/create'
   */
-  checkForPreviousRouteForRedirect() {
-    const previousUrlObj = this.navigationHelperService.getPreviousUrl();
-    if (previousUrlObj && previousUrlObj.url && (previousUrlObj.url !== '/workspace/content/create')) {
-      this.redirect();
-    }
+ checkForPreviousRouteForRedirect() {
+  const previousUrlObj = this.navigationHelperService.getPreviousUrl();
+  if (previousUrlObj && previousUrlObj.url &&
```

Comment:
change this condition to minimize the code and maximize the readability
```
let url = _.get(previousUrlObj, 'url')
let  routes =[ '/workspace/content/create', '/workspace/content/create/training']
if(url && _.indexOf(routes, url) === -1){
     this.redirect();
}
```

Reply:
done